### PR TITLE
Cross platform support

### LINKED ISSUE #5: New password gen core
Excerpt from another issue:

I think then we definitely want our own implementation. The crypto+rng part is especially important as we need this to be portable to react native. Crypto there must be done on the native side and transferred over an async channel to the JS.. so this library needs to expose a RNG override.

I would also suggest the following options, which could be rendered as checkboxes in some UI:

- Uppercase letters
- Lowercase letters
- Digits
- Spaces
- Underscores / Dashes
- Symbols (basic keyboard range, no slashes)
- Add from custom set (specified with an input)

I would suggest that all of these be hardcoded - regex is horribly inefficient at this task and should be avoided.

### PR DESCRIPTION
**Pull request by perry-mitchell**
Fixes #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,30 +1,19 @@
...
   "dependencies": {
-    "mngen": "^1.0.0",
-    "password-generator": "^2.2.0"
+    "deep-freeze": "0.0.1",
+    "random-number-generator": "^1.0.2"
   },
   "devDependencies": {
-    "ava": "^0.23.0",
-    "husky": "^0.14.3",
-    "lint-staged": "^4.3.0",
```
Comment (sallar) [inline package.json]: Can we add back husky and lint-staged and prettier?

Comment (perry-mitchell) [reply package.json]: Yep! Still WIP sorry 😅

Comment (sallar) [reply package.json]: Ah looked final to me sorry :D

**Comment by perry-mitchell** [comment]
Ready for another review @sallar

**Review by sallar — APPROVED** [review]
Great work!

**Pull request merged**